### ISSUE TITLE
Missing aria-labels

### ISSUE THREAD

**Issue by SaiShan** [SAP/fundamental-ngx#915]
#### Is this a bug, enhancement, or feature request?
Bug (accessibility)

#### Briefly describe your proposal.
If a user tabs to the objects, the user must know what element he has selected and how many other items are in the current group. (ie. "Option 1, item 1 of 4")
       - Combobox input
       - Dropdown
       - Menu
       - Side navigation
